### DISCUSSION TITLE
这里有全量镜像源可供快速同步

### DISCUSSION THREAD

**Discussion by miaomiaotail** [bangbang93/openbmclapi#58] [General]
地址lineapp.mycloudnas.com
端口21
协议FTPs（TLS1.3）
用户名openbmclapi
密码openbmclapi123
路径/
限速：无
Mode Z压缩：有（需要选择与Mode Z相容的、Unicode编码的客户端）
访问日志：详细（此日志会被开放）
联系：此issue下面

**Comment by miaomiaotail** [comment]
这个FTP服务器会详细记录访问日志

**Comment by weyeahh** [comment]
不好用
